package.json

```diff
@@ -58,11 +58,11 @@
     "geckodriver": "1.19.1",
     "http-server": "0.12.3",
     "jsdom": "16.2.2",
-    "minify": "^5.0.0",
+    "minify": "^5.1.1",
     "mocha": "^7.1.0",
     "np": "6.2.3",
     "prismjs": "1.20.0",
-    "replace": "1.2.0",
+    "replace": "1.1.5",
```

Comment:
Waarom was dit nodig?

Reply:
De nieuwste versie veroorzaak problemen bij onze replaces:

Verwacht resultaat:
``` JavaScript
import { analytics } from '/node_modules/vl-ui-cookie-consent/src/analytics.js';
import '/node_modules/vl-ui-cookie-consent/dist/vl-cookie-consent-opt-in.js';
```

Resultaat na upgrade:
``` JavaScript
import { analytics } from /node_modules/vl-ui-cookie-consent/src/analytics.js;
import /node_modules/vl-ui-cookie-consent/dist/vl-cookie-consent-opt-in.js;
```

Er lijkt dus een probleem te zijn met quotes. Ik kon het probleem oplossen door deze wijziging door te voeren:

``` JavaScript
replace(`${quoted('/src/vl-(.*)')}`, `'/node_modules/vl-ui-${this.webcomponent}/dist/vl-\\$1'`, file); // previous code
replace(`${quoted('/src/vl-(.*)')}`, `'/node_modules/vl-ui-${this.webcomponent}/dist/vl-\\$1'`, file); // code with space fix
```

Dergelijke fix wou ik liever niet doen. Ik vermoed dus een probleem in de replace dependency, maar heb (nog) geen tijd geïnvesteerd om het daar te onderzoeken en issue of PR te maken, voorlopig alleen fixed versie toegepast als fix.